tests/test_0001_sdk_version_and_default_conn.py

```diff
@@ -23,14 +23,18 @@ def test_version(test_env: str) -> None:  # pylint: disable=unused-variable
         )
         with open(filename, "r", encoding=sys.getdefaultencoding()) as f:
             version = f.readline()
-            # assert watchful.__version__ \
-            #     == importlib.metadata.version(_get_package_name()) \
-            #     == version, f"versions are {watchful.__version__}, " \
-            #         f"{importlib.metadata.version(_get_package_name())} and " \
-            #         f"{version}."
-            assert (
-                watchful.__version__ == version
-            ), f"Versions are different: {watchful.__version__} and {version}."
+            # assert (watchful.__version__
```

Comment:
Do we need these commented lines?

Reply:
Thanks for checking with me on this. They use features available from Python 3.8 such as `importlib.metadata`; I'm using them back in the code since we are committing to Python 3.8.12.